assets/src/components/vpn/graphql/mutations.ts

```diff
@@ -0,0 +1,17 @@
+import { gql } from 'apollo-boost'
+
+const CreateWireguardPeer = gql`
+  mutation CreateWireguardPeer($email: String, $name: String!, $userId: ID) {
+    createPeer(email: $email, name: $name, userId: $userId) {
+      raw
```

Comment:
I think you can query the status and other fields here now w/o it erroring

Reply:
Yeah, but I am not using the result of create mutation in any way right now.